Analyzer/plugins/Analyzer.cc

```diff
@@ -988,7 +990,7 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
     matchedMuonWasFound = true;
     const reco::Muon* triggerObjMatchedMu = &(muonColl)[closestTrigMuIndex];
     // To prove that it's fine to use the official SFs
-    if (fabs(triggerObjMatchedMu->eta()) < 1.0) {
+    if (fabs(triggerObjMatchedMu->eta()) < 2.4) {
```

Comment:
we should make this use the configurable variable for eta (which already exists)